translations/sv/home.json

```
@@ -0,0 +1,20 @@
+{
+  "intro_what_is_fediverse": "Vad är <mark>fediversumet</mark>?",
+  "intro_btn_explore": "Utforska",
+  "intro_analogy": "En bra jämförelse för fediversumet är <mark>e-post</mark>.",
+  "intro_email": "Du använder förmodligen Gmail. Eller kanske Outlook. Eller Yahoo Mail.",
+  "intro_anyone": "Oavsett ditt val kan du skicka ett e-postmeddelande till <mark>vem som helst</mark> i världen utan att behöva veta eller bry dig om vilken tjänst de använder.",
```

```suggestion
  "intro_anyone": "Oavsett vad du väljer kan du skicka ett e-postmeddelande till <mark>vem som helst</mark> i världen utan att behöva veta eller bry dig om vilken tjänst de använder.",
```
Change to active tense.